app/models/Assessment.scala

```diff
@@ -51,9 +60,52 @@ object Assessment {
       PropertyAddress.fromString(valuationHistory.address),
       valuationHistory.billingAuthorityReference,
       valuationHistory.currentFromDate,
-      valuationHistory.currentToDate,
-      capacityDeclaration
+      valuationHistory.currentToDate
+    )
+  }
+
+  def fromValuationHistory(valuationHistory: ValuationHistory, authorisationId: Long) = {
+    Assessment(
+      authorisationId,
+      valuationHistory.asstRef,
+      valuationHistory.listYear,
+      valuationHistory.uarn,
+      valuationHistory.effectiveDate.getOrElse(LocalDate.now()),
```

Comment:
what do we use effective date for?

Reply:
We use it in the DvrController.alreadySubmittedDetailedValuationRequest
If i pass an option all the way through I can handle it in the view, like I said in my previous comment.

Reply:
Passed through as option and handled in the frontend